src/data/markdown/docs/03 cloud/03 Integrations/05 Cloud APM.md

```diff
@@ -11,10 +11,11 @@ k6 Cloud platform supports exporting metrics to APM platforms, thereby enabling
 
 Each supported APM platform is called a provider in Cloud APM. As you'll see in each platform's respective section, the `provider` is a key passed to the APM configuration object and its value should match the providers listed below. Also, each provider has a separate set of configuration parameters. Therefore you need to visit your provider's page:
 
-| Provider       | Platform page                                                | Supported Regions                                                                        |
-| -------------- | ------------------------------------------------------------ | ---------------------------------------------------------------------------------------- |
-| `datadog`      | [DataDog](/cloud/integrations/cloud-apm/datadog)             | [DataDog supported regions](/cloud/integrations/cloud-apm/datadog#supported-regions)     |
-| `azuremonitor` | [Azure Monitor](/cloud/integrations/cloud-apm/azure-monitor) | [Azure supported regions](/cloud/integrations/cloud-apm/azure-monitor#supported-regions) |
+| Provider       | Platform page                                                                    | Supported Regions                                                                        |
+| -------------- | -------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `datadog`      | [DataDog](/cloud/integrations/cloud-apm/datadog)                                 | [DataDog supported regions](/cloud/integrations/cloud-apm/datadog#supported-regions)     |
+| `azuremonitor` | [Azure Monitor](/cloud/integrations/cloud-apm/azure-monitor)                     | [Azure supported regions](/cloud/integrations/cloud-apm/azure-monitor#supported-regions) |
+| `prometheus`   | [Prometheus Remote Write](/cloud/integrations/cloud-apm/prometheus-remote-write) | Not applicable                                                                           |
```

Comment:
Will DataDog and Azure Monitor also move under prometheus page as well in a future iteration? Seems like we shouldn't have separate workflows for them if it's not needed? This is more of a generic feedback than a change request.

Reply:
I am not sure about this and haven't investigated it myself. The way those integration are developed are different (and more mature) than Prometheus. It may change in the future, but then the product team should decide. @sniku @robingustafsson

Reply:
Then it's good to go. We can revisit that discussion at a later point if necessary. 👍🏼